src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -128,3 +129,13 @@ Napi::Value ModelWrap::is_dynamic(const Napi::CallbackInfo& info) {
     const auto result = _model->is_dynamic();
     return Napi::Boolean::New(env, result);
 }
+
+Napi::Value ModelWrap::get_output_size(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    if (info.Length() > 0) {
+        reportError(env, "getOutputSize() does not accept any arguments.");
```

Comment:
```suggestion
        reportError(env, "get_output_size() does not accept any arguments.");
```
Should error message use same name current function?

Reply:
No, because it shows at JS side. From JS side it has `getOutputSize` name.